ocaml/xapi/xapi_xenops.ml

```diff
@@ -1507,7 +1507,8 @@ let update_vm ~__context id =
 							(fun (_, state) ->
 								let gm = Db.VM.get_guest_metrics ~__context ~self in
 								debug "xenopsd event: Updating VM %s PV drivers detected %b" id state.pv_drivers_detected;
-								Db.VM_guest_metrics.set_PV_drivers_detected ~__context ~self:gm ~value:state.pv_drivers_detected
+								Db.VM_guest_metrics.set_PV_drivers_detected ~__context ~self:gm ~value:state.pv_drivers_detected;
+								Db.VM_guest_metrics.set_PV_drivers_up_to_date ~__context ~self:gm ~value:state.pv_drivers_detected
```

Comment:
No, but PV_drivers_up_to_date is a deprecated field and it's logic even before this commit was nothing to do with "up to date". See https://github.com/xapi-project/xen-api/blob/master/ocaml/idl/datamodel.ml#L7428.